.github/workflows/build.yml

```diff
@@ -7,15 +7,24 @@ on:
     branches: ["main"]
 
 jobs:
-  build:
+  rust-stable:
```

Comment:
This job is named a bit weirdly

Reply:
do you have any better ideas? :trollface: